unit2.js

```diff
@@ -0,0 +1,79 @@
+
+//1. Setup the button for the DOM to listen
+document.addEventListener('DOMContentLoaded', () => {
+    setupFormListener();
+})
+//2. Setup for the form listener
+const setupFormListener = () => {
+    //console.log("it's listening")
+    let form = document.querySelector('form')
+    form.addEventListener('submit', (event) => {
+        event.preventDefault();
+    })
+}
+
+//3. upload film title to the menu selection options
+const createFilmList = () => {
+    const url = "https://ghibliapi.herokuapp.com/films/"
+    fetch(url)
+        .then(response => response.json())
+        .then(data => {
+            let filmMenu = document.querySelector('#movie-titles')
+
+            let startOption = document.createElement('option')
+            startOption.text = "Choose a Movie"
+
+            filmMenu.add(startOption);
+            filmMenu.selectedIndex = 0;
+
+            let movieName;
+            for (let i = 0; i <= data.length; i++) {
+                movieName = document.createElement('option');
+                movieName.innerText = data[i].title;
+                filmMenu.add(movieName);
+            }
+            selectedMovie();
```

Comment:
This function should run when the user chooses one of thee options from select tag through event listener on "change".